src/main/java/survey/OpenEndedQuestion.java

```diff
@@ -1,10 +1,24 @@
 package survey;
```

Comment:
Everyone, we should implement a format where we comment on top of each class stating what the class does and its relevance to the entire project

Reply:
Yeah I agree

Reply:
I'm sure there is autoformatter for having fields to be inputted at the top of the class. Something like: 
" Description:
  Contributors:
  ....etc"

If not we can create our own template for classes.